src/lib/components/molecules/canvas-map/lib/styles/Text.js

```diff
@@ -1,6 +1,60 @@
+/**
+ * The placement of the text relative to its position on the map
+ * @readonly
+ * @enum {string}
+ * @property {string} TOP - The top of the text is placed closest to the anchor.
+ * @property {string} BOTTOM - The bottom of the text is placed closest to the anchor.
+ * @property {string} LEFT - The left side of the text is placed closest to the anchor.
+ * @property {string} RIGHT - The right side of the text is placed closest to the anchor.
+ * @property {string} CENTER - The center of the text is placed closest to the anchor.
+ * @property {string} TOP_LEFT - The top left corner of the text is placed closest to the anchor.
+ * @property {string} TOP_RIGHT - The top right corner of the text is placed closest to the anchor.
+ * @property {string} BOTTOM_LEFT - The bottom left corner of the text is placed closest to the anchor.
+ * @property {string} BOTTOM_RIGHT - The bottom right corner of the text is placed closest to the anchor.
+ *
+ * @example
+ * const textAnchor = TextAnchor.CENTER;
+ */
+const TextAnchor = {
+  TOP: "top",
+  BOTTOM: "bottom",
+  LEFT: "left",
+  RIGHT: "right",
+  CENTER: "center",
+  TOP_LEFT: "top-left",
+  TOP_RIGHT: "top-right",
+  BOTTOM_LEFT: "bottom-left",
+  BOTTOM_RIGHT: "bottom-right",
+}
```

Comment:
Nitpick: I tend to prefer string unions over enums like this: we don't have to import anything to provide a value, and it's easier to troubleshoot runtime issues (e.g. when we see a stray `top-left`, we don't have to know that it came from `TextAnchor.TOP_LEFT`, it'll appear as `top-left` in our source code)

Reply:
Forgot to read this before I hit merge, sorry. Could you give me an example of what that would look like? I'm not sure I understand.

Reply:
No worries! Not a blocking issue at all.

It'd look something like this...

```js
/** @typedef { "top" | "bottom" | "left" | "right" | "center" | "top-left" | ... } TextAnchor */
```

Then further down, you'd just use this type as the parameter type...

```js
export class Text {
  /**
   * Create a text element style
   * @constructor
   * @param {TextStyle} options - Style options
   */
  constructor(options) {
    this.content = options?.content
    this.anchor = options?.anchor || "center"
    // ...
  }
}
```

When dealing with these `TextAnchor` values, you'll still get completion, and Typescript will still shout at you if you provide the wrong value.

<img src="https://github.com/user-attachments/assets/16c1bad8-1a23-4edd-9513-7c219f35deec" width=500>
<img src="https://github.com/user-attachments/assets/85c3f6c1-1410-4cb1-a0aa-663d81f57ee2" width=500>

Reply:
Nice one, thanks! That's a much cleaner solution. I'll use it from now on.